src/ai/backend/agent/server.py

```diff
@@ -812,15 +821,28 @@ async def server_main(
 @click.option(
     "--debug",
     is_flag=True,
-    help="Enable the debug mode and override the global log level to DEBUG.",
+    help="This option will soon change to --log-level TEXT option.",
+)
+@click.option(
+    "--log-level",
+    type=click.Choice(LogLevel, case_sensitive=False),
+    default="info",
+    help="Choose logging level from... debug, info, warning, error, critical",
 )
 @click.pass_context
 def main(
     cli_ctx: click.Context,
     config_path: Path,
-    debug: bool,
+    log_level: str,
+    debug: bool = False,
 ) -> int:
 
+    if debug:
+        click.echo("Please use --log-level options instead")
+        click.echo("--debug options will soon change to --log-level TEXT option.")
+        log_level = "debug"
```

Comment:
`log_level` is `LogLevel` type
```suggestion
        log_level = LogLevel.DEBUG
```

if you want to assign a string value to `log_level`, you need to declare another variable here.